bindings/py/nupic/bindings/regions/TestNode.py

```diff
@@ -299,8 +302,11 @@ def getProtoType():
 
 
   def write(self, proto):
-    pass
-
+    regionImpl = proto.regionImpl.as_struct(TestNodeProto)
+    regionImpl.int32Param = self.getParameter('int32Param', 0)
```

Comment:
double quotes. Also, any reason to wait on the other parameters? Should be trivial to add them here.